src/secondary.rs

```diff
@@ -228,4 +245,102 @@ where
 
 impl<'a, K> FusedIterator for BitVecIter<'a, K> where K: TryFrom<usize> {}
 
-// TODO: Implementations for HashSet, BTreeSet, HashMap, BTreeMap.
+impl<K> SecondaryMap<K, bool> for HashSet<K>
+where
+    K: Hash + Eq + Clone,
+{
+    type Iter<'a> = HashSetIter<'a, K> where Self: 'a, K: 'a;
+
+    #[inline]
+    fn new() -> Self {
+        HashSet::new()
+    }
+
+    #[inline]
+    fn with_capacity(capacity: usize) -> Self {
+        HashSet::with_capacity(capacity)
+    }
+
+    #[inline]
+    fn default_value(&self) -> bool {
+        false
+    }
+
+    #[inline]
+    fn ensure_capacity(&mut self, capacity: usize) {
+        HashSet::reserve(self, capacity.saturating_sub(self.capacity()));
+    }
+
+    #[inline]
+    fn resize(&mut self, _new_len: usize) {}
```

Comment:
not sure I follow why this is empty

Reply:
The method was meant to truncate the linear vecs in Unmanaged map and BitVec, but that doesn't make sense on sparse unordered containers.

It should probably be removed from the trait (it's never used).